runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -1045,7 +1045,7 @@ MM_StandardAccessBarrier::preMountContinuation(J9VMThread *vmThread, j9object_t
 		/* concurrent scavenger in progress */
 		MM_EnvironmentStandard *env = MM_EnvironmentStandard::getEnvironment(vmThread->omrVMThread);
 		MM_ScavengeScanReason reason = SCAN_REASON_SCAVENGE;
-		_scavenger->getDelegate()->scanContinuationNativeSlots(env, contObject, reason);
+		_scavenger->getDelegate()->scanContinuationNativeSlots(env, contObject, reason, true);
```

Comment:
for easier reading introduce a line before:
const bool beingMounted = true;

same for RT Barrier